src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -21,6 +21,7 @@
     <IncludeDebugSymbolsInLocalVSIXDeployment>true</IncludeDebugSymbolsInLocalVSIXDeployment>
     <CopyBuildOutputToOutputDirectory>true</CopyBuildOutputToOutputDirectory>
```

Comment:
Yes. CopyBuildOutputToOutputDirectory to false would probably break the fast-up-to-date check. You'd also want the PDB in the local deployment for debugging that hive as well, since this project does still produce code that runs.